evap/staff/templates/staff_semester_todo.html

```diff
@@ -0,0 +1,55 @@
+{% extends "staff_semester_base.html" %}
+
+{% load i18n %}
+{% load staff_templatetags %}
+{% load morefilters %}
+
```

Comment:
here the text for the breadcrumbs is missing. it's using ```{% block subtitle %}``` - please have a look at e.g. ```staff/templates/staff_semester_base.html```